core/templates/dev/head/base.html

```diff
@@ -225,10 +225,44 @@
                             </ul>
                           </li>
                         {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if-start="windowWidth > 768"></create-exploration-button>
```

Comment:
Yeah, Bootstrap causes the problem. I assumed we were allowed to use ng-if-start, So I didn't answer No.